### ISSUE TITLE
Feature Request: Keyword Arguments

### ISSUE THREAD

**Issue by adam-grant-hendry** [spyder-ide/qtpy#349]
C++ doesn't natively support keyword arguments, but Python does. Calls to `Qt` functions from `PyQt` return a "no matching signature" or "not enough arguments" error when keywords are used.

Would it be possible to wrap `Qt` calls in the python bindings so keyword arguments are supported?

This may not be possible (or appropriate) to do within `qtpy` directly. I submitted a feature request to the PySide issue tracker here:

[PYSIDE-1964](https://bugreports.qt.io/browse/PYSIDE-1964)

If it is better to have `Qt` implement this, feel free to monitor the above request ticket. Hopefully it will be noticed.

**Comment by dalthviz** [comment]
Hi @adam-grant-hendry thank you for the feedback! I'm not sure what we could do over th qtpy side to help with this sorry :/ However, could you please post a code example with the situation with kwargs you are describing just to be sure we have a clear understanding of the feature you want? Thanks!

**Comment by CAM-Gerlach** [comment]
Yeah, I'm pretty sure this would need to be done at the binding level if its feasible; my main concern would be that it would make it much more difficult to ensure compatibility between the bindings and versions without a ton of fragile shimming, if one binding implements it but not the other, it isn't backported to Qt5, or they implement it in an incompatible way (e.g. different argument names).
